src/script_opt/ScriptOpt.cc

```diff
@@ -26,7 +26,6 @@ AnalyOpt analysis_options;
 std::unordered_set<const Func*> non_recursive_funcs;
 
 void (*CPP_init_hook)() = nullptr;
-void (*CPP_activation_hook)() = nullptr;
```

Comment:
So this already wasn't used anymore?

Reply:
Yeah, seems this in-tree user isn't needed anymore. I've pinged Vern about it. If the `CPP_activation_hook` is required down the road, can use the plugin hook instead.